azurerm/resource_arm_media_services_test.go

```diff
@@ -0,0 +1,298 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/mediaservices/mgmt/2018-07-01/media"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func init() {
+	resource.AddTestSweepers("azurerm_media_services", &resource.Sweeper{
+		Name: "azurerm_media_services",
+		F:    testSweepMediaServicesAccount,
+	})
+}
+
+func testSweepMediaServicesAccount(region string) error {
```

Comment:
We can also remove the sweeper functions, azure rm does not used them anymore.

Reply:
Removed.